finality-aleph/src/party.rs

```diff
@@ -543,14 +551,15 @@ pub(crate) fn create_aleph_config(
     n_members: usize,
     node_id: NodeIndex,
     session_id: SessionId,
+    base_unit_creation_delay: u64,
```

Comment:
Can you change the type to `BaseUnitCreationDelay`? the float should be unwrapped as late as possible, so that its meaning is reflected by the type for as long as possible. Ideally, I would see this in the `exponential_slowdown`, but this would require changing public interface, so maybe not?